lib/diagnostic.rb

```diff
@@ -100,13 +108,15 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = arr.slice!(2..3)
```

Comment:
Haven't seen slice! work before. Plus in looking at the documentation, it calls for something else: like `slice_when`, `slice_after`, etc.:

`arr.pop(2)` would work just fine here.